docs/source/concepts/local-ledger.rst

```diff
@@ -0,0 +1,296 @@
+In particular, an application can process all subsequent events from the flat transaction stream or the transaction tree stream without having to take events before the snapshot into account.
+
+A deployed DAML ledger ensures that all the local ledgers of its Participant Nodes are consistent, subject to the ledger's trust assumptions.
+Here, we consider only a single DAML ledger.
+The general case of :ref:`several interoperable DAML ledgers <interoperable-ledgers>` is discussed :ref:`elsewhere <interoperable-causality>`.
+Consistency can be expressed by embedding all the local ledgers into a single graph of transactions, the shared ledger.
+A shared ledger satisfies conditions similar to a local ledger, but the conditions do not restrict the orders to a party.
+
+Definition »shared ledgers«
+  A **shared ledger** is a finite directed acyclic graph that satisfies the following conditions:
+
+  * *Total order on contract changes:*
+    For every contract, the action order totally orders all activations and deactivations of the contract.
+    In this sequence, activations alternate with deactivations, starting with an activation.
+
+  * *Total order on key changes:*
+    For every key, the action order totally orders all allocations and deallocations of the key.
+    In this sequence, allocations alternate with deallocations, starting with an allocation, and allocation-deallocations pairs refer to the same contract.
+
+  * *Use ordering:*
+    If an action `act`:sub:`u` uses a contract,
+    then `act`:sub:`u` must be ordered after an activation action of the contract.
+    Let `act`:sub:`a` be the last activation action of the contract before `act`:sub:`u`.
+    Then there must not be a deactivation action `act`:sub:`d` that is ordered after `act`:sub:`a` and before `act`:sub:`u`.
+    
+    If an action `act`:sub:`u` uses a key
+    and there is a key allocation action `act`:sub:`a` ordered before `act`:sub:`u`,
+    then there is a deallocation action `act`:sub:`d` for the key ordered after `act`:sub:`a` and before `act`:sub:`u`.
+
+Definition »order consistency«
+  A set `Ls` of local ledgers is **order-consistent** for a single DAML Ledger
+  if there exists a shared ledger `G` with the following properties:
+
+  * Whenever `Ls` contains a local ledger `L` for party `A`,
+    the vertices of `L` are exactly the vertices of `G` projected to `A` unless the projection is empty.
+    (This takes multiplicities into account, i.e., if the projections of two vertices in `G` are the same,
+    `L` contains two vertices with the same transactions and it is fixed which vertex in `L` belongs to which vertex in `G`.)
+    Moreover, whenever `L` contains an edge between two vertices, `G` contains an edge between the corresponding vertices in `G`.
+
+Order consistency between local ledgers ensures that the order of transactions is *consistent* across several Participant Nodes and between parties.
```

Comment:
Since we have a completely different notion of consistency in the ledger model, I'd suggest "order coherency" here, to avoid confusion.